packages/builders/scripts/fast-usdc/fast-usdc-reconfigure.build.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Usage:
+ * agoric run fast-usdc-reconfigure.build.js [--agoricToNoble <IBCChannelInfo>]
+ */
+import { makeHelpers } from '@agoric/deploy-script-support';
+import { getManifestForUpdateNobleICA } from '@agoric/fast-usdc/src/update-noble-ica.core.js';
+import { IBCConnectionInfoShape } from '@agoric/orchestration';
+import { mustMatch } from '@endo/patterns';
+import { parseArgs } from 'node:util';
+
+/**
+ * @import {CoreEvalBuilder, DeployScriptFunction} from '@agoric/deploy-script-support/src/externalTypes.js'
+ * @import {IBCConnectionInfo} from '@agoric/orchestration'
+ * @import {Passable} from '@endo/marshal'
+ */
+
+/**
+ * @param {Parameters<CoreEvalBuilder>[0]} powers
+ * @param {{ agoricToNoble?: IBCConnectionInfo } & Passable} options
+ * @satisfies {CoreEvalBuilder}
+ */
+export const proposalBuilder = async ({ publishRef, install }, options) => {
+  return harden({
+    sourceSpec: '@agoric/fast-usdc/src/update-noble-ica.core.js',
+    /** @type {[string, Parameters<typeof getManifestForUpdateNobleICA>[1]]} */
+    getManifestCall: [
+      getManifestForUpdateNobleICA.name,
+      {
+        options,
+        installKeys: {
+          fastUsdc: publishRef(
+            install('@agoric/fast-usdc/src/fast-usdc.contract.js'),
+          ),
+        },
+      },
+    ],
+  });
+};
+
+/** @type {DeployScriptFunction} */
+export default async (homeP, endowments) => {
+  const { writeCoreEval } = await makeHelpers(homeP, endowments);
+
+  /** @returns {{ agoricToNoble?: IBCConnectionInfo } & Passable} */
+  const parseConnection = () => {
```

Comment:
usually these builder functions are almost declarative.

consider moving this up to module scope,
```js
export const parseConnection = (args) => {
```

that would also allow unit testing